esphome/components/stepper/__init__.py

```diff
@@ -82,6 +92,8 @@ async def setup_stepper_core_(stepper_var, config):
         cg.add(stepper_var.set_deceleration(config[CONF_DECELERATION]))
     if CONF_MAX_SPEED in config:
         cg.add(stepper_var.set_max_speed(config[CONF_MAX_SPEED]))
+    if CONF_ROTATION in config:
+        cg.add(stepper_var.set_rotation(config[CONF_ROTATION]))
```

Comment:
As there is a default, we dont need to check
```suggestion
    cg.add(stepper_var.set_rotation(config[CONF_ROTATION]))
```